consumer/consumer.go

```diff
@@ -54,7 +55,7 @@ const (
 type Consumer interface {
 	Serve()
 	Close() error
-	ProcessMessage(msg *sarama.ConsumerMessage) error
+	ProcessMessage(msg *sarama.ConsumerMessage) (interface{}, error)
```

Comment:
that's pretty untyped :)

Reply:
Didn't realize it was possible to use `string` when unmarshaling `null` and missing fields, but apparently it's fine, so we no longer have to use this ugly trick.